api/turing/config/example.yaml

```diff
@@ -28,6 +28,19 @@ DeployConfig:
   MaxCPU: "4"
   MaxMemory: "8Gi"
 
+# New Relic Config
+NewRelic:
+  Enabled: true
+  AppName: turing
+  License: <your very secret license key here>
+  IgnoreStatusCodes: 
+    - 400
```

Comment:
What is the rationale for ignoring these codes?

Reply:
This is just an example.